core/electron-manager/src/frontend/ElectronApp.ts

```diff
@@ -10,8 +10,9 @@ import { ElectronRpcManager } from "../common/ElectronRpcManager";
 
 /**
  * Frontend Ipc support for Electron apps.
+ * @beta
  */
-class ElectronIpc implements IpcSocketFrontend {
+export class ElectronIpc implements IpcSocketFrontend {

```

Comment:
@kckst8 I don't understand why you need to export this. Nobody should need it

Reply:
is this just about Auth config? Let me look a better way to do that.

Reply:
@kabentley yes, auth config for this case. Basically any configuration that is cached in the backend and may be needed pre-startup.

Reply:
Well that would only those that are used *during* startup. Otherwise they're not need pre-startup, by definition. I think it's only Auth config, and i think the whole thing is weird because the backend reads it, sends it to the frontend so it can send it to the backend. That's nuts.

Basically, i think we need an new approach to the whole concept of "config" and there shouldn't be a `Config` object in the frontend at all. 

@wgoehrig @calebmshafer can we get together to discuss?

Reply:
@kckst8 please see [this PR](https://github.com/imodeljs/imodeljs/pull/1101). It is a breaking change and will need lots of testing, but i think it's worth doing.